### ISSUE TITLE
Compile error when running run_flow.sh

### ISSUE THREAD

**Issue by wuruoyu** [tum-ei-eda/utvm_staticrt_codegen#11]
Hi, Firstly thank you for this great project. 

I install the package following [CI Config](https://github.com/tum-ei-eda/utvm_staticrt_codegen/blob/master/.github/workflows/ci.yml) and it works well. 

Then I run the ```run_flow.sh``` script, an error happens. The error message is below:
Thanks in advance!

```
### TVMFlow.loadModel
Input "dense_2_input" float32 (1, 1)
### TVMFlow.build
One or more operators have not been tuned. Please tune your model for better performance. Use DEBUG logging level to see more details.
__nop function is not yet supported.__nop function is not yet supported.__nop function is not yet supported.op[1]: __nop
  num args: 2
    sz arg: 4  [0x42cec0]
    sz arg: 4  [0x42cec0]
op[4]: tvmgen_default_fused_nn_contrib_dense_pack_add_nn_relu_1
  num args: 4
    sz arg: 4  [0x42cec0]
    sz arg: 64  [0x430ec0]
    sz arg: 64  [0x424ec0]
    sz arg: 64  [0x42e6c0]
op[5]: __nop
  num args: 2
    sz arg: 64  [0x42e6c0]
    sz arg: 64  [0x42e6c0]
op[8]: tvmgen_default_fused_nn_contrib_dense_pack_add_nn_relu
  num args: 4
    sz arg: 64  [0x42e6c0]
    sz arg: 1024  [0x42dec0]
    sz arg: 64  [0x4306c0]
    sz arg: 64  [0x42fec0]
op[9]: __nop
  num args: 2
    sz arg: 64  [0x42fec0]
    sz arg: 64  [0x42fec0]
op[12]: tvmgen_default_fused_nn_contrib_dense_pack_add
  num args: 4
    sz arg: 64  [0x42fec0]
    sz arg: 64  [0x42d6c0]
    sz arg: 4  [0x42eec0]
    sz arg: 4  [0x42e6c0]
-- The C compiler identification is Clang 9.0.1
-- The CXX compiler identification is Clang 9.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib/ccache/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/lib/ccache/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/target_src/build
[  6%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/apps/bundle_deploy/bundle_static.c.o
[ 12%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/common/crt_backend_api.c.o
[ 18%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/common/func_registry.c.o
[ 25%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/common/crt_runtime_api.c.o
[ 37%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/common/packed_func.c.o
[ 37%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/common/ndarray.c.o
[ 43%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/memory/page_allocator.c.o
[ 50%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/memory/stack_allocator.c.o
[ 56%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/graph_executor/load_json.c.o
[ 62%] Building C object CMakeFiles/tvm_static_rt.dir/home/ruoyu/lib/microTVM_aot/tvm/src/runtime/crt/graph_executor/graph_executor.c.o
[ 68%] Linking C static library libtvm_static_rt.a
[ 68%] Built target tvm_static_rt
[ 75%] Building C object CMakeFiles/example_target_src.dir/main.c.o
[ 81%] Building C object CMakeFiles/example_target_src.dir/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c.o
[ 87%] Building C object CMakeFiles/example_target_src.dir/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/default_lib0.c.o
[ 93%] Building C object CMakeFiles/example_target_src.dir/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/default_lib1.c.o
/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c:45:5: warning: implicit declaration of function 'TVMPlatformAbort' is invalid in C99 [-Wimplicit-function-declaration]
    TVMPlatformAbort(kTvmErrorPlatformCheckFailure);
    ^
/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c:52:5: warning: implicit declaration of function 'TVMPlatformAbort' is invalid in C99 [-Wimplicit-function-declaration]
    TVMPlatformAbort(kTvmErrorPlatformCheckFailure);
    ^
/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c:59:5: warning: implicit declaration of function 'TVMPlatformAbort' is invalid in C99 [-Wimplicit-function-declaration]
    TVMPlatformAbort(kTvmErrorPlatformCheckFailure);
    ^
/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c:117:6: error: conflicting types for 'TVMPlatformAbort'
void TVMPlatformAbort(tvm_crt_error_t code) {
     ^
/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c:45:5: note: previous implicit declaration is here
    TVMPlatformAbort(kTvmErrorPlatformCheckFailure);
    ^
3 warnings and 1 error generated.
make[2]: *** [CMakeFiles/example_target_src.dir/build.make:90: CMakeFiles/example_target_src.dir/home/ruoyu/lib/microTVM_aot/utvm_staticrt_codegen/examples/out/staticrt.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:85: CMakeFiles/example_target_src.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

**Comment by rafzi** [comment]
Thank you for the interest and the report.

This should fix the issue: https://github.com/tum-ei-eda/utvm_staticrt_codegen/commit/1b296f1efcc28671ee21ccbe86a81b968db8b1e5

**Comment by PhilippvK** [comment]
@rafzi Oops I also fixed it in 407673ec0a94a0c273a60b50867efeb2419ef506 but was waiting for the CI to complete :D

**Comment by wuruoyu** [comment]
Thanks! It works very well!
